source/private.rst

```diff
@@ -35,15 +34,19 @@ backups involves following steps:
    for backups. Otherwise, this setting may be omitted. You may also notice the
    ``MINIO_REGION`` value which is may not be used within a private
    cloud. Use the same region value here and on later steps
-   (``us-east-1`` is a good default choice).
+   (``us-west-1`` is a good default choice).
```

Comment:
us-east-1 is a typical default, why is `us-west-1` better?

Reply:
It's more like synchronization between our doc file and cr.yaml. cr.yaml by default comes with us-west-1 region, which I can't change in this PR.

Reply:
Could we just change CR, not doc in this case? I prefer `us-east-1`

Reply:
ok